java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java

```diff
@@ -196,11 +196,11 @@ private Future<List<ConsumerRecord>> poll() {
                         response.body().forEach(obj -> {
                             JsonObject json = (JsonObject) obj;
                             list.add(new ConsumerRecord(
-                                json.getString("topic"),
-                                json.getValue("key"),
-                                json.getValue("value"),
-                                json.getInteger("partition"),
-                                json.getLong("offset"))
+                                    json.getString("topic"),
+                                    json.getValue("key"),
+                                    json.getValue("value"),
+                                    json.getInteger("partition"),
+                                    json.getLong("offset"))
```

Comment:
I guess this change is unrelated to the PR, can we avoid it please?

Reply:
@ppatierno I carried some old commits from a rebase. Fixing now